packages/instant/src/components/payment_method.tsx

```diff
@@ -107,4 +116,24 @@ export class PaymentMethod extends React.Component<PaymentMethodProps> {
                 );
         }
     };
+    private readonly _handleInstallWalletClick = (): void => {
```

Comment:
This is feeling kind of unnatural to me -- triggering `onInstallWalletClick` only when we are on desktop.  

If we keep this name, I would think we would trigger something like `onInstallWalletClick(walletType: 'coinbaseWallet' | 'metamask')`.  

If we want to keep this pattern, I think we should rename this to something like `onInstallMetamaskClick` or `onInstallDesktopWalletClick`

Reply:
I think its fine since we already have mobile specific logic when rendering, I think we can either move all the mobile specific logic out and up into a higher component or keep it as is

Reply:
Oh I actually changes this.